packages/design-system/src/hooks/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { useClipboard } from 'native-base'
+import { useCallback } from 'react'
+import { Toast } from '../atoms'
+
+export const useCopyOnClipboard = () => {
```

Comment:
Let's rename the hook to be more explicit.

```suggestion
export const useClipboardWithToast = () => {
```